src/jqLite.js

```diff
@@ -284,7 +284,7 @@ function JQLite(element) {
   }
   if (!(this instanceof JQLite)) {
     if (argIsString && element.charAt(0) !== '<') {
-      throw jqLiteMinErr('nosel', 'Looking up elements via selectors is not supported by jqLite! See: http://docs.angularjs.org/api/angular.element');
+      return new JQLite(document.querySelector(element));
```

Comment:
Thanks for letting me know.
I've been busy this entire time but I'll take the time to finish the PR.